config/config.exs

```diff
@@ -0,0 +1,13 @@
+use Mix.Config
+
+config :membrane_core, Membrane.Logger,
```

Comment:
Why do you configure the logger here? It only generates clutter while running tests

Reply:
I wanted to test how logs look (wether they are formatted as I inteded) and forgot to remove it.